Orange/widgets/classify/owsvmclassification.py

```diff
@@ -1,17 +1,15 @@
-import numpy as np
 from collections import OrderedDict
 
 from PyQt4 import QtGui
 from PyQt4.QtCore import Qt
 
 from Orange.data import Table
-from Orange.classification.svm import SVMLearner, SVMClassifier, NuSVMLearner
-from Orange.widgets import widget, settings, gui
-from Orange.widgets.utils.owlearnerwidget import OWProvidesLearner
-from Orange.widgets.utils.sql import check_sql_input
+from Orange.classification.svm import SVMLearner, NuSVMLearner
+from Orange.widgets import settings, gui
+from Orange.widgets.utils.owlearnerwidget import OWBaseLearner
 
 
-class SVMBaseMixin(OWProvidesLearner):
+class SVMBaseMixin(OWBaseLearner):
```

Comment:
SVMBaseMixin is now used as a sole parent of SVM widgets, so Mixin could be dropped from the name.